docs/src/tutorials/exporting.md

```diff
@@ -1,51 +1,173 @@
+To download your entire app's code history:
 
-## Download data from services
+{{< codetabs >}}
+
+---
+title=Using the CLI
+file=none
+highlight=false
+---
+
+1. List all your projects with:
+
+   ```bash
+   platform projects
+   ```
+
+2. Retrieve the project you want to back up with:
+
+   ```bash
+   platform get {{< variable "PROJECT_ID" >}}
+   ```
+
+<--->
+
+---
+title=Using Git
+file=none
+highlight=false
+---
+
+1. In the [Console](https://console.platform.sh/), open your project and click **Code {{< icon chevron >}}**.
+2. Click **Git**.
+3. To copy the command, click **{{< icon copy >}}**.
+4. To clone the repository, run a command similar to the following:
+
+   ```text
+   git clone abcdefgh1234567@git.eu.platform.sh:abcdefgh1234567.git project-name
+   ```
+
+{{< /codetabs >}}
+
+## 2. Download your files
+
+Some files might not be stored in Git,
+such as data your app writes [in mounts](../create-apps/app-reference.md#mounts).
+
+To download your files:
+
+{{< codetabs >}}
+
+---
+title=Using the CLI
+file=none
+highlight=false
+---
+
+1. Get a list of all your mounts with:
+
+   ```bash
+   platform mount:list
+   ```
+
+2. Download all of your files from a given mount with:
+
+   ```bash
+   platform mount:download --mount {{< variable "MOUNT_PATH" >}} --target ./{{< variable "LOCAL_FOLDER" >}}
+   ```
+
+<--->
+
+---
+title=Using SSH
+file=none
+highlight=false
+---
+
+You can transfer files through SSH using `scp` or `rsync`.
+
+{{< /codetabs >}}
+
+For more examples, see how to [transfer files to and from a built app](../development/file-transfer.md).
+
+## 3. Download data from services
 
 The mechanism for downloading from each service (such as your database) varies.
-For services designed to hold non-persistent information (such as Redis or Solr) it's generally not necessary to download data as it can be rebuilt from the primary data store.
 
-To download data from persistent services ([MySQL](../add-services/mysql/_index.md), [PostgreSQL](../add-services/postgresql.md), [MongoDB](../add-services/mongodb.md), or [InfluxDB](../add-services/influxdb.md)), see each service's page for instructions.
+For services designed to hold non-persistent data, such as [Redis](../add-services/redis.md) or [Solr](../add-services/solr.md),
+it's generally not necessary to download data as it can be rebuilt from the primary data store.
+
+For services designed to hold persistent data, see each service's page for instructions:
+
+- [MySQL](../add-services/mysql/_index.md#exporting-data)
+- [PostgreSQL](../add-services/postgresql.md#exporting-data)
+- [MongoDB](../add-services/mongodb.md#exporting-data)
+- [InfluxDB](../add-services/influxdb.md#exporting-data)
+
+## 4. Get the environment variables
+
+Environment variables can contain critical information such as tokens or additional configuration options for your app.
+
+Environment variables beginning with:
```

Comment:
:lady_beetle: As with code blocks, the introductory sentence should stand alone: https://developers.google.com/style/lists#introductory-sentences-for-lists The current phrasing is rather confusing for a list.